test/acceptance/test/testsuite_common_test.go

```diff
@@ -44,9 +44,9 @@ var _ = BeforeSuite(func() {
 	GITHUB_ORG = os.Getenv("GITHUB_ORG")
 	WEGO_BIN_PATH = os.Getenv("WEGO_BIN_PATH")
```

Comment:
Curious if keeping this env var the same was intentional. Not a huge deal either way.

Reply:
I had to decide where to draw the line in the tests and stopped at actual command uses and the comments in the ginkgo tests. I wasn't sure about this one either.